data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
-			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`
-
-
-
-# The B mission is for players that kept the Remnant secret from the Deep, but did the wormhole research.
-mission "Remnant: Follow-up on Terminus Researchers B"
-	name "Check up on Wormhole Researchers"
-	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
-	minor
-	source
-		government "Remnant"
-	destination "Valhalla"
-	to offer
-		has "Remnant: Learning Sign 2: done"
-		has "Deep: Wormhole Research: offered"
-		not "Remnant: Follow-up on Terminus Researchers A: done"
-		not "Remnant: Follow-up on Terminus Researchers B: active"
-		random < 40
-	on offer
+			
+			label admitted2
+			`	"This is good," Torza responds. "Once you have the findings, simply return to the spaceport here and give them to the information desk. We will look over them shortly thereafter." Torza gets up from the bench and shows you out. "Have a safe journey, Captain. But know that we may be keeping a closer eye on you going forward to avoid another situation like this."`
+				accept
+			
+			label revealed2
+			branch liar2
+				or
+					has "remnant: lied to torza: technology"
+					has "remnant: lied to torza: understanding"
+			
+			`	"This is good," Torza responds. "Once you have the findings, simply return to the spaceport here and give them to the information desk. We will look over them shortly thereafter." Torza gets up from the bench and shows you out without another word.`
+				accept
+			
+			label liar2
+			apply
+				set "remnant: player is surveilled"
+			`	"This is good," Torza responds. "Once you have the findings, simply return to the spaceport here and give them to the information desk. We will look over them shortly thereafter." Torza gets up from the bench and shows you out. "Know that you will be watched closely by us from now on, Captain. I commend you for agreed to bring us this information, but we can not allow another situation like this to occur."`
+				accept
+	
+	# Suspicion == 0: Get everything
+		# surveillance
+	# Suspicion == 1: Get exactly what's asked for no trouble
+		# first time, knew after, brought tech
+	# Suspicion == 2: Get what's asked for, slight suspicion
+		# first time, knew after, no tech || first time, knew before, brought tech
+	# Suspicion == 3: Get old info, suspicious
+		# first time, knew before, no tech || second time, knew before, brought tech
+	# Suspicion == 4: Confronted, get nothing
+		# second time, knew before, no tech
```

Comment:
Thank you for the suspicion numbers cheat-sheet. Very helpful!

Reply:
Given that I'd constructing the suspicions by adding together various conditions, I can't really take a single scenario and say "this one gets a slight bump or drop" without having even more branching and apply nodes to account for such scenarios. I'd rather just leave things as they are, especially because I wrote Ivan's responses according to these scenarios.